ticket.html

```diff
@@ -111,6 +112,151 @@ <h2>Log in</h2>
             </div>
         </header>
         <main class="page">
+            <div class="page__ticket">
+                <div class="ticket__container _container">
+                    <div class="ticket__title">Select your seats for booking tickets</div>
+                    <div class="ticket__block">
+                        <div class="ticket__display">
+                            <img src="img/cinema_line.png" alt="">
+                        </div>
+                        <div class="section__list">
+                            <div class="section" id="A" onClick="gettingID(this.id,event)">
```

Comment:
A is a very generic name and name like this can make your code more hard to read and trace the possible bugs
```suggestion
                            <div class="section" id="section_A" onClick="gettingID(this.id,event)">
```